[Content] Quarg Acknowledges Player Flying An Arfecta And Warns Them (#7688)
endless-sky/endless-sky: data/quarg/quarg missions.txt

			label end
			`	Before you can ask it anything else, the Quarg bids you farewell, and walks away.`
				decline

mission "Quarg Pug Arfecta Warning"
	landing
	to offer
		has "First Contact: Quarg: offered"
		has "flagship model: Pug Arfecta"
	source
		attributes "quarg"

> 		attributes "station"

Reviewer: why only on stations?

Author: At first I just copied the "Ask Quarg About Coalition Early" mission to get started.

I looked at that  source condition and thought maybe I should remove it, but then I thought there was more narrative weight if they warned you about the power of the ship, a ship that was most likely used to destroy one of their rings, on one of their rings.

Reviewer 2: There is a `"ringworld"` attribute for that, which differs from stations

Author: Gotcha will make that change. I guess the station in human space doesn't count as a ring world because the ring isn't done yet.

	on offer
		conversation

> 			`As soon as you step off your ship a rather imposing Quarg blocks your exit ramp while stairing piercingly at your ship.`

Reviewer 3: ```suggestion
			`As soon as you step off your ship a rather imposing Quarg blocks your exit while staring piercingly at your ship.`
```

No need to impose a ramp on the Arfecta here where it's not actually relevant.

> 			`"It is as though a baby in swaddling has killed a dragon, and taken its treasure for its own..." it says with a mix of amusement and disgusted.`

Reviewer 3: ```suggestion
			`"It is as though a baby in swaddling has killed a dragon, and taken its treasure for its own..." it says with a mix of amusement and disgust.`
```

> 			`"It is as though a baby in swaddling has killed a dragon, and taken its treasure for its own..." it says with a mix of amusement and disgusted.`

Reviewer 3: ```suggestion
			`"It is as though a baby in swaddling has killed a dragon, and now rides its corpse, thinking to have mastered fire..." it says with a mix of amusement and disgust.`
```
A more fitting analogy, imo, for both the situation and the ship. You haven't looted something the dragon guards, you have desecrated and weapon used the dragon's own body.

Author: Would changing "killed a dragon" to "tamed a dragon" work too?

So 

"It is as though a baby in swaddling has tamed a dragon, and now rides it through the sky spewing fire with reckless abandon."

			choice
				`	"It's a long story..."`
					goto evasive
				`	"I prefer to frame it as David and Goliath myself."`
					goto righteous
				`	"The Pug fancy themselves gods, but God is dead, soo..."`
					goto anarchy
				`	"I saw something I liked, so I made it mine."`
					goto greedy
			label evasive
			`	"It matters not how it happened."`
				goto end
			label righteous

> 			`	"Did not your Jesus compare himself to this David also?" the Quarg says with what you assume is a sarcastic tone.`

Reviewer 3: ```suggestion
			`	"Did not your Jesus compare himself to this David also?" the Quarg says with what you assume is a sarcastic tone.`
```
This one hits me squarely wrong in many ways.

It demonstrates the Quarg have a specific understanding of a single religion that runs deeper than their grasp of the language as a whole, but also implies that they understand -only- such one specific religion; they are responding as one believer to another, not as an outsider. Even then, the phrasing here, more than the others, implies the player is a believer as well, which is entirely inappropriate.

This would be much better either as a response with a parable from a differing religion (the more different, the better) or simply outright ignorance of the reference.

Would be best served, imo, with replacing the initial reference entirely -in addition- to the above.

Reviewer 2: I gotta agree with that, lets avoid references to actual earth religions.
Myths are fine though

Reviewer 2: I don't know if I want the ES universe to have that religion cannon, especially in that state and with the same names, so yeah I'd prefer a more generic response, especially since not everyone will know about who this is and what they did if we use a specific story.
Also best not to open the can of worm of religious things, some may interpret it badly and whatnot, no need to risk it

Author: Would a reference to Hercules work? It’s from a religious body of work, just one that is no longer practice and has passed into myth.

Or should I aim for more generic than that?

Author: Updated this path to be more generic let me know if that works.

				goto end
			label anarchy

> 			`	"And you believe yourself in a position to judge?" the Quarg says irritated.`

Reviewer 3: ```suggestion
			`	"And you believe yourself in a position to judge?" the Quarg says irritated.`
```

This really doesn't follow from the lead in to me. Not even just thematically, no judgement was being passed.

I'd suggest something more directly calling out the arrogant hypocrisy in the line this one is responding to.

Author: Would the following work?

"So you took it upon yourself to prove them wrong?" the Quarg says with an amused incredulous smile.

Given that the Quarg may actually agree that the Pug think themselves god, but would still be annoyed with your arrogance.

Reviewer 3: Yea, that would work very well there I think.

				goto end
			label greedy

> 			`	"A baby in swaddling as I said before."`

Reviewer 3: ```suggestion
			`	"A baby in swaddling, as I said before."`
```

				goto end
			label end
			`	"We give you a warning. You know not what you have child, the weapon you wield."`

> 			`	"We have seen its power. We have felt its power. Be careful that you are this ship's captain, not its thrall."`

Reviewer 3: ```suggestion
			`	"We have seen its power. We have felt its power. Be careful that you are this ship's captain, not its thrall."`
```

This could do with a major tone shift away from the ship/weapon itself being such a threat and towards chastising the impetuous and ignorant nature of the acquisition.

Author: Was going for less the ship being a threat, and more a lesser being in possession of it being a threat to the balance of the galaxy.

Would this help?

"We have seen its power. We have felt its power. We have learned from its power. We have now even overcome its power. But you and your kind know nothing of this power and its burdens and dangers. Be careful that you are this ship's captain, not its thrall."

Reviewer 3: Even with this angle I'd go a bit further with the end.

"Be careful that you direct this power, and not it you."

Could even go so far as to add after that "And remember that we have learned to overcome it."

Author: I like that

Author: Updated with suggestion

			label end
			`	Before you can ask it anything else, the Quarg bids you farewell, and walks away.`
				decline

mission "Quarg Pug Arfecta Warning"

> 	landing

Reviewer 2: ```suggestion
	minor
	landing
```

I think this ought to be a minor thing that wont happen in the middle of a story

	to offer
		has "First Contact: Quarg: offered"
		has "flagship model: Pug Arfecta"
	source
		attributes "quarg"
		attributes "ringworld"
	on offer
		conversation
			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at your ship.`

> 			`"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon," it says with a mix of amusement and disgust.`

Reviewer 2: ```suggestion
			`	"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon," it says with a mix of amusement and disgust.`
```
Pretty sure we need to indent everything that is not in the first section of ``

> 			`"It is as though a baby in swaddling has tamed a dragon, and now rides it through the sky spewing fire with reckless abandon." it says with a mix of amusement and disgust.`

Reviewer 3: ```suggestion
			`"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon." it says with a mix of amusement and disgust.`
```

One more little slip in the rewriting it looks like.

			choice
				`	"It's a long story..."`
					goto evasive

> 				`	"I prefer to frame it as an act of righteous defiance."`

Reviewer 2: ```suggestion
				`	"It was an act of righteous defiance."`
```
mb better like this? Caus otherwise it feels like its not smth you say but a description of smth you say to me
Especially considering the Quarg's response

					goto righteous

> 				`	"The Pug fancy themselves gods, but God is dead, soo..."`

Reviewer 2: I love that reference to Nietzsche
Otherwise I'd have said to change it to "I don't believe in Gods"
But since its a ref, keep it!

Reviewer 2: ```suggestion
				`	"The Pug fancy themselves gods, but since God is dead..."`
```
Would it be better that way? Caus idk about the so....

					goto anarchy
				`	"I saw something I liked, so I made it mine."`
					goto greedy
			label evasive

> 			`	"It matters not how it happened."`

Reviewer 2: I think that this line and maybe all others could be missing the usual Quarg elements of visual depictions of their body?
Or at least some more description, like `he scoffs, "it matters not how it happened"`

The Quarg body things happening is valid for all others phrases too I think, the Quarg would not be willing to speak much about it but you should be able to see all kinds of emotions? The same kind we get when we show the broken ring world fragment

Author: Sounds good. Will make the change

				goto end
			label righteous
			`	“You fancy yourself a savior or a prophet then?” the Quarg says with what you assume is a sarcastic tone.`
				goto end
			label anarchy

> 			`	"So you took it upon yourself to prove them wrong?" the Quarg says with an amused incredulous smile.`

Reviewer 3: ```suggestion
			`	"So you took it upon yourself to prove them wrong?" the Quarg says with an amused, incredulous smile.`
```

Or even explicitly "and" instead of the comma. Dealer's choice :)

				goto end
			label greedy
			`	"A baby in swaddling, as I said before."`
				goto end
			label end
			`	"We give you a warning. You know not what you have child, the weapon you wield."`
			`	"We have seen its power. We have felt its power. We have learned from its power. But you and your kind know nothing of this power and its burdens and dangers. Be careful that you direct this power, and not it you."`
			`	"And be careful to remember that we have learned to overcome it."`

> 			`	The Quarg then curtly motions farewell and leaves you to your business. As you go you feel the crowd's glare follow you.`

Reviewer 2: they motion farewell? would they really do that, in this context?

Author: Like a dismissive hand motion from the center down and out. Like see ya and a get out of here motion

Author: Ideas welcome most definitely welcome here.

Reviewer 2: Maybe they just suddenly turn back, as a way of saying this conversation is over?
All Quargs could turn around Klingon style but that'd be a bit too much, though just one could be nice

Reviewer 4: @Arachi-Lover Thoughts?

Reviewer 5: You could have the Quarg clearly signal the end of the conversation with the dismissive motion or such, and then explicitally take one more, long look at the Arfecta. The crowd can then be said to disperse in a few minutes, but a few Quarg remain, gawking at the player character and/or the Arfecta.

Author: Updated this line. I think the current version addresses all feedback.

			label end
			`	Before you can ask it anything else, the Quarg bids you farewell, and walks away.`
				decline

mission "Quarg Pug Arfecta Warning"
	minor
	landing
	to offer
		has "First Contact: Quarg: offered"
		has "flagship model: Pug Arfecta"
	source
		attributes "quarg"
		attributes "ringworld"
	on offer
		conversation

> 			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at your ship.`

Reviewer 6: ```suggestion
			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather around. As you go to disembark, you find a rather imposing Quarg blocking the exit while staring piercingly at your ship.`
```

> 			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at your ship.`

Reviewer 7: ```suggestion
			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at you.`
```
"Piercingly" strikes me as a strange word to describe the Quarg's gaze with respect to your ship because it begs the question, "what exactly are they piercing?" (The ship's hull?) I typically think of "piercing" used to describe how one person looks at another (i.e. "staring piercingly through you"), such that they are piercing obfuscations or lies or feigned ignorance, all of which the imposing Quarg demonstrates later in the conversation.

Author: Makes sense will update myself as this line now has a conflict.

Author: Change made

			`	"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon," it says with a mix of amusement and disgust.`
			choice
				`	"It's a long story..."`
					goto evasive
				`	"It was an act of righteous defiance."`
					goto righteous
				`	"The Pug fancy themselves gods, but since God is dead..."`
					goto anarchy
				`	"I saw something I liked, so I made it mine."`
					goto greedy
			label evasive
			`	It scoffs, "It matters not how it happened."`
				goto end
			label righteous
			`	“You fancy yourself a savior or a revolutionary then?” the Quarg says with what you assume is a sarcastic tone.`
				goto end
			label anarchy
			`	"So you took it upon yourself to prove them wrong?" the Quarg says with an amused, incredulous smile.`
				goto end
			label greedy
			`	"A baby in swaddling, as I said before."`
				goto end
			label end

> 			`	"We give you a warning. You know not what you have child, the weapon you wield."`

Reviewer 6: ```suggestion
			`	"We give you a warning. You know not what you have, child - the weapon you wield."`
```

> 			`	"We have seen its power. We have felt its power. We have learned from its power. But you and your kind know nothing of this power and its burdens and dangers. Be careful that you direct this power, and not it you."`

Reviewer 7: ```suggestion
			`	"We have seen its power. We have felt its power. We have learned from its power. But you and your kind know nothing of this power and its burdens and dangers. Be careful that you are the one who directs this power, and not it you."`
```
This came across as somewhat hard to parse on my first reading, so I think that making the syntax more explicit could be beneficial.

> 			`	"And be careful to remember that we have learned to overcome it."`

Reviewer 6: ```suggestion
			`	"And be careful to remember that we have learned how to overcome it."`
```

> 			`	The Quarg curtly motions farewell then takes one last long look at your ship before leaving you to your business. The crowd too starts to disperse, with a few staying behind to gawk at your ship.`

Reviewer 6: ```suggestion
			`	The Quarg curtly motions farewell then takes one last long look at your ship before leaving you to your business. The crowd too starts to disperse, with a few staying behind to gawk at it.`
```

> 			`	As you make your way through the ring you notice all the Quarg onboard glaring at you with disgust.`

Reviewer 6: ```suggestion
			`	As you make your way through the ring, you notice that all the Quarg onboard glare at you with disgust.`
```

> 			`	As you make your way through the ring you notice all the Quarg onboard glaring at you with disgust.`

Reviewer 7: ```suggestion
			`	As you make your way through the ring you notice many of the Quarg onboard glaring at you with disgust.`
```
This feels like a stronger response than is set up by the rest of the conversation, especially considering the previous line has the stragglers of the crowd "gawking" at your ship in awe rather than regarding you with disgust. Is the crowd composed only of non-Quarg?

Author: Makes sense will update directly because of a line conflict

Author: Change made

	on offer
		conversation
			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at your ship.`

> 			`"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon." it says with a mix of amusement and disgust.`

Reviewer 8: ```suggestion
			`"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon," it says with a mix of amusement and disgust.`
```

	on offer
		conversation "quarg imprisonment"

mission "Quarg Pug Arfecta Warning"
	minor
	landing
	to offer
		has "First Contact: Quarg: offered"
		has "flagship model: Pug Arfecta"
	source
		attributes "quarg" "ringworld"
	on offer
		conversation
			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather around. As you go to disembark, you find a rather imposing Quarg blocking the exit while staring piercingly at you.`
			`	"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon," it says with a mix of amusement and disgust.`
			choice
				`	"It's a long story..."`
					goto evasive
				`	"It was an act of righteous defiance."`
					goto righteous
				`	"The Pug fancy themselves gods, but since God is dead..."`
					goto anarchy
				`	"I saw something I liked, so I made it mine."`
					goto greedy
			label evasive
			`	It scoffs, "It matters not how it happened."`
				goto end
			label righteous

> 			`	“You fancy yourself a savior or a revolutionary then?” the Quarg says with what you assume is a sarcastic tone.`

Reviewer 9: ```suggestion
			`	"You fancy yourself a savior or a revolutionary then?" the Quarg says with what you assume is a sarcastic tone.`
```
Swap fancy quote for normal one.

	on offer
		conversation "quarg imprisonment"

mission "Quarg Pug Arfecta Warning"
	minor
	landing
	source

> 		attributes "quarg" "ringworld"

Reviewer 10: Multiple attributes on the same line means "A or B". In this case, we want this to offer on a planet that is both Quarg AND a ringworld, in which case we use separate attribute lines.
```suggestion
		attributes "quarg"
		attributes "ringworld"
```

				goto choice
			label flagship
			`As soon as you land your ship, a crowd of Quarg start to gather around. As you go to disembark, you find a rather imposing Quarg blocking the exit. Tall even by Quarg standards, it towers over you and gives you a piercing stare.`
			label choice
			`	"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon," it says while not breaking its gaze, unblinking.`
			choice
				`	"It's a long story..."`
					goto evasive
				`	"It was an act of righteous defiance."`
					goto righteous
				`	"The Pug fancy themselves gods, but since God is dead..."`
					goto anarchy
				`	"I saw something I liked, so I made it mine."`
					goto greedy
			label evasive
			`	It scoffs, "It matters not how it happened.`
				goto end
			label righteous
			`	"You fancy yourself a savior or a revolutionary then?" the Quarg says with what you assume is a sarcastic tone.`
				goto end
			label anarchy

> 			`	"So you took it upon yourself to prove them wrong?" the Quarg says with an amused, incredulous smile.`

Reviewer 11: Definitely optional, but I wonder if this sounds better?

```suggestion
			`	"So you deign to prove them wrong?" the Quarg says with an almost amused, or perhaps incredulous tone.`
```

	on offer
		conversation "quarg imprisonment"

mission "Quarg Pug Arfecta Warning"
	minor
	landing
	source
		attributes "quarg"
		attributes "ringworld"
	to offer
		has "First Contact: Quarg: offered"
		has "ship model: Pug Arfecta"
	on offer
		conversation
			branch flagship
				has "flagship model: Pug Arfecta"

> 			`As soon as you land, you see a crowd of Quarg gathering around one of your escorts, the Pug Arfecta. The captain of the ship looks shaken as the Quarg surround them, seemingly being questioned by a rather tall Quarg, even by Quarg standards. The captain looks around and, seeing you, points in your direction, causing the Quarg to come your way. The tall Quarg towers over you and gives you a piercing stare.`

Reviewer 2: What if this is your flagship?

Reviewer 10: Then it goes to the label from the branch right above this line.

Reviewer 2: Oh yeah I'm blind, phone review ig
Sorry
